### DISCUSSION TITLE
Notes vs. Annotations

### DISCUSSION THREAD

**Discussion by brianebeling** [aidenlx/obsidian-zotero#127] [Help]
Zotero can annotate pdfs (e.g. with ZotFile) but there is also the option "Add Note from Annotations"
For some reason these notes are getting ingored by the plugin when importing a literature note. 

The annotations in the pdf show up, however.

The question now is two-fold:
- Will this plugin support "notes" or only annotations?
- Am I doing something wrong when promoting annotations to notes?

I'd love to gain some insight on how you are tackling the issue, maybe I missunderstood.

![context menu](https://i.imgur.com/kDB6eJJ.png)

The red marked notes don't show up
![notes](https://i.imgur.com/3ZMPqY5.png)

Some random annotation to test does show up correctly
![annotation pane works](https://i.imgur.com/7RJe9lb.png)

**Comment by asan-emirsaleh** [comment]
I recently encountered the same issue too. I have created annotations (i.e. added some text to coloured highlights) via Zotero pdf browser and notes via Zotero notes interface located in the right side pannel. As a result of import, there were no notes transferred, but the annotations were successfully imported.

**Comment by aidenlx** [comment]
Both are supported in v2. Annotations come from the PDF as before, and Zotero notes written on an item — including ones made with **Add Note from Annotations** — are converted from Zotero's HTML to Markdown and written as their own files, linked from the literature note. Nothing gets silently skipped anymore.

How the import works: https://zotlit.aidenlx.site/docs/how-to/import-zotero-notes

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
